saleor/discount/models.py

```diff
@@ -55,22 +54,30 @@ def active_in_channel(self, date, channel_slug: str):
         )
 
     def expired(self, date):
-        return self.filter(
-            Q(used__gte=F("usage_limit")) | Q(end_date__lt=date), start_date__lt=date
+        return self.filter(end_date__lt=date, start_date__lt=date)
+
+
+class VoucherManager(models.Manager):
+    def get_queryset(self):
```

Comment:
I would add typing here: `def get_queryset(self) -> VoucherQueryset` to satisfy linter as it expects standard `QuerySet`

Reply:
I dont need this class, I will revert this changes